wg-batch/annual-report-2022.md

```diff
@@ -5,9 +5,8 @@
 1. What work did the WG do this year that should be highlighted?
    For example, artifacts, reports, white papers produced this year.
 
-   -
-   -
-   -
+   - We're still gathering feedback from multiple users running various batch-related frameworks and projects on kubernetes, see [our agenda](https://docs.google.com/document/d/1XOeUN-K0aKmJJNq7H07r74n-mGgSFyiEDQ3ecwsGhec/edit) for both past and upcoming topics.
+   - We've closely collaborated with SIG Apps, SIG Node and SIG Scheduling to improve batch workloads on kubernetes.
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
```

Comment:
Development of Kueue, 2 releases in 2022

Reply:
Added